controllers/apply/lib/field-types/file.js

```diff
@@ -74,11 +69,12 @@ class FileField extends Field {
 
     get displayValue() {
         if (this.value) {
-            const mimeType = mime.extension(this.value.type) || 'File';
-            const fileSize = filesize(this.value.size, { round: 0 });
-            return `${
-                this.value.filename
-            } (${mimeType.toUpperCase()}, ${fileSize})`;
+            const mimeType = mime.extension(this.value.type);
```

Comment:
Reworked this display logic so we don't show "FILE" in english only if we try to format an unknown file type.